### ISSUE TITLE
Error could not find semver

### ISSUE THREAD

**Issue by boswelja** [r0adkll/sign-android-release#27]
I've just started getting the following error when using this:
`Error: Could not find file '/home/runner/work/_actions/_temp_048ce8a4-bb03-49f4-855e-1f0b02f1614b/_staging/r0adkll-sign-android-release-d28dc26/node_modules/jest-snapshot/node_modules/.bin/semver'.`
I believe ff34f681c113c5965a5b44406bf0033326f29886 is the culprit, it appears semver was completely removed.

**Comment by seanwu1105** [comment]
Having the same error.

```
Download action repository 'r0adkll/sign-android-release@v1'
Error: Could not find file '/home/runner/work/_actions/_temp_a2cdbd55-181b-4601-b50e-54e5c932c6e6/_staging/r0adkll-sign-android-release-d28dc26/node_modules/jest-snapshot/node_modules/.bin/semver'.
```

The workflow file:

``` yml
- name: Sign release AAB
  uses: r0adkll/sign-android-release@v1
  with:
    releaseDirectory: ./android/app/build/outputs/bundle/release
    signingKeyBase64: ${{ secrets.ANDROID_KEYSTORE_FILE }}
    alias: ${{ secrets.ANDROID_KEY_ALIAS }}
    keyStorePassword: ${{ secrets.ANDROID_KEYSTORE_PASSWORD }}
    keyPassword: ${{ secrets.ANDROID_KEY_PASSWORD }}
```

This action works fine before, but it broke this two hours ago.

**Comment by VladSumtsov** [comment]
Having the same issue. 
```
Download action repository 'r0adkll/sign-***-release@v1'
Error: Could not find file '/home/runner/work/_actions/_temp_ce5c90bd-16c3-46b8-8588-43c824c9cae0/_staging/r0adkll-sign-***-release-d28dc26/node_modules/jest-snapshot/node_modules/.bin/semver'.
```

The workflow file:
```
      - name: sign release build
        uses: r0adkll/sign-android-release@v1
        with:
          releaseDirectory: app/build/outputs/apk/release/
          signingKeyBase64: ${{ secrets.SIGNING_KEY_BASE64 }}
          keyStorePassword: ${{ secrets.KEYSTORE_PASSWORD }}
          alias: ${{ secrets.KEY_ALIAS }}
          keyPassword: ${{ secrets.KEY_PASSWORD }}
          signedReleaseFile: app/build/outputs/apk/release/app_release.apk
```

This action works fine yesterday and broke today.

**Comment by tmorone-rezi** [comment]
Use this until it gets fixed:
uses: r0adkll/sign-android-release@21072f7435f485b65555103ff2c60e86560b759e

Adding release tags to stable versions would be a good practice.
Thanks for maintaining this package!

**Comment by lhache** [comment]
I confirm what @tmorone-rezi said. We did the same and it worked.

Meanwhile, there is an attempt at fixing this in #29 
I cross fingers that update jest & co would help.

I also noticed that node_modules are committed, I hope to understand why (cf #28)

**Comment by r0adkll** [comment]
okay, I've merged your PR, updated dependencies, re-built src, and made sure the latest `node_modules` were added. That should hopefully resolve this issue. I have also tagged and released this under `v1.0.3` so @tmorone-rezi you can start referring to stable versions/tags. 

I appreciate everyone's patience and I'm sorry for the inconvenience! (Typescript/Web is not my main discipline 😆 )

**Comment by boswelja** [comment]
The issue with semver is resolved, but now I'm getting an issue with missing 'node-which'
Error here: `Error: Could not find file '/home/runner/work/_actions/_temp_5f3633f9-22f8-4199-babb-6da7ae92880c/_staging/r0adkll-sign-android-release-6079bfc/node_modules/cross-spawn/node_modules/.bin/node-which'.`
